inference-engine/src/plaidml_plugin/ops/reorg_yolo.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("reorgYolo", [](const Context& ctx) {
+  auto* layer = ngraph::as_type<ngraph::opset4::ReorgYolo>(ctx.layer);
+  IE_ASSERT(ctx.operands.size() == 1);
+  auto I = ctx.operands.at(0);
+  auto strides = layer->get_strides();
+  return edsl::make_tuple(op::reorg_yolo(I, strides[0], false));
```

Comment:
after check the ops.cc in plaidml, reorg_yolo op's 'decrease' means that 'C*stride*stride' or 'C/(stride*stride)'.
according openvino reorg_yolo spec , we should set 'false' here.